exports['axios error interceptor should add context in error object for failed get request 1'] = {
  "name": "AxiosError",
  "code": "ERR_BAD_REQUEST",
  "status": 404,
  "context": {
    "response": {
      "status": 404,
      "statusText": null,
      "headers": {},
      "data": ""
    },
    "method": "get"
  },
  "response": {
    "status": 404,
    "statusText": null,
    "headers": {},
    "data": ""
  }
}
